README.md

```diff
@@ -17,6 +17,18 @@ See also the "Alternatives" section in this README.
 Changelog
 ---------
 
+### Upcoming Version
+
+- Added support for the Java 7 `X509ExtendedTrustManager` if your project uses Java 7 please use `Java7Pinning` instead of `JavaPinning`.
+- Two new factory methods `JavaPinning.trustManagerForPins(String...)` and `JavaPinning.forPins(String...)`.
+- Deprecated `JavaPinning.trustManagerforpins()` in favor of the correctly named `JavaPinning.trustManagerForpins()`.
+- Two new factory methods `Pin.fromCertificate` and `Pin.fromPublicKey` these methods can be used to directly create pins from X509 certificates or X509 public keys.
```

Comment:
Just spotted I forgot to update the change-log for the `fromCertificate` and `fromPublicKey` method change, will do this when home!